dds/DCPS/TopicImpl.h

```diff
@@ -87,9 +87,15 @@ class OpenDDS_Dcps_Export TopicImpl
 
   /**
    * If the representation sequence of a reader or writer is empty then fill it
-   * either from the QoS of the topic or what the type allows.
+   * from the QoS of the topic.
    */
```

Comment:
The QoS should not be inherited like this. It turned out that was the users job, like it is with the rest of the QoS. I don't think I said that explicitly in my proposed changes though.

Reply:
Does it need to happen in QoS_Helper::copy_from_topic_qos (or is that done already)?

Reply:
Yes, but I don't remember if it's already doing it or not.

Reply:
It is already there, yes. 

I'll remove the inherit function.